### DISCUSSION TITLE
Benchmarking against other software

### DISCUSSION THREAD

**Discussion by LeifParr** [petrobras/ross#911] [Q&A] [answered]
Hi all.

I recently started in a new company, and before letting me perform rotor dynamic analyses using Ross, they want me to prove that the software is sufficient by performing a benchmark test versus other commercially available software. My problem is that I do not have access to the commercial software (MADYN), as this is purchased by another branch of our company.
I tried duplicating a MADYN-run based on manual transfer of a MADYN input file model to a Ross model, but as I lack some important overall parameters (global inertia, global mass), I am not able to get a good match on lateral natural frequencies.

I have seen that there is a publication on Ross on research-gate, regarding general aspects of the software. Does anyone know if there has been published a benchmarking (Ross performance versus other commercial software) article somewhere, that I could use as a reference?

**Comment by raphaeltimbo** [comment, marked as answer]
Hi @LeifParr. 
We do not have an article with these benchmarks.
But we did carry out some comparisons against other software, in particular XLTRC.
In the comparison, we used a centrifugal compressor model and the coefficients for bearings and seals were calculated beforehand by a different software. So, the comparison was on ROSS capabilities of assembling the correct system matrices and providing good results in analysis such as damped critical speed (Campbell Diagram), unbalance response, etc.
These comparisons were carried out with an old version of the code. If you would like, maybe we could run these comparisons again with the current code and add them to the documentation.

**Reply by LeifParr** [reply]
I believe this would be great. I did internal benchmarking versus RAPPID at my previous work, where I also had a licence for RAPPID, but alas, these results are left at the previous job. This benchmarking versus RAPPID had very similar results between RAPPID / Ross.

**Comment by LeifParr** [comment]
[RAPPID-output-Ross-example-rotor-500rpm.txt](https://github.com/ross-rotordynamics/ross/files/10002170/RAPPID-output-Ross-example-rotor-500rpm.txt)

For reference, I ran the example_rotor() through RAPPID to compare natural frequencies at 500 rpm shaft speed. The RAPPID output is attached. The first modes are well matched, both in frequency and mode shape.

**Reply by raphaeltimbo** [reply]
Great @LeifParr. Can I add this in our documentation for reference?

**Comment by LeifParr** [comment]
Yes, I do not believe that there should be any copyright issues or such. We
could refer to the software package RAPPID, and more specifically the RAPP
module. The software vendor web site is https://www.rda.guru/
